depclean-gradle-plugin/src/test/resources/all_dependencies_used/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
remove wrapper folder.

Reply:
Ya, I think we can remove these resources wrapper folders.